ui/src/shared/components/threesizer/Division.tsx

```diff
@@ -77,7 +83,14 @@ class Division extends PureComponent<Props> {
           <div className={this.titleClass}>{name}</div>
         </div>
         <div className={this.contentsClass} style={this.contentStyle}>
-          {name && <div className="threesizer--header" />}
+          {name && (
```

Comment:
Might think about adding a getter that can convey more information about why the name is important for this component to show.